cypress/integration/group1/checkerWorkflowFromTools.ts

```diff
@@ -24,6 +24,7 @@ describe('Checker workflow test from my-tools', () => {
   });
 
   function goToB3() {
+    cy.wait(1000);
```

Comment:
:o

Reply:
Oh ya...I put those waits as temporary fixes.  Should do a real fix for it at some point.

Reply:
Might be a good idea to do it now while we're early in a release and not under the gun for a release date